### PR TITLE
refactor(package.json & contributing.md): add pnpm link.dist script

### PR DESCRIPTION
**Pull request by maiieul**
# What is it?

- [x] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

First PR :nerd_face: 

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by maiieul** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by maiieul** [comment]
recheck

File: package.json
Comment (maiieul) [inline package.json:1]: just re-organized by alphabetical order and added "link.dist"

**Comment by thejackshelton** [comment]
Looks good to me! Merging.

The CI error seems to be a problem with GitHub and the "danger bot" for first time contributors.

**Comment by gioboa** [comment]
@all-contributors please add @maiieul for code, ideas, maintenance

**Pull request merged**